aptos-move/e2e-move-tests/tests/code_publishing.data/code_publishing_framework_upgrade_fail.exp

```diff
@@ -14,13 +14,13 @@ Ok(
                             AccessPath(
                                 AccessPath { address: 0000000000000000000000000000000000000000000000000000000000000001, path: 01000000000000000000000000000000000000000000000000000000000000000104636f696e09436f696e53746f7265010700000000000000000000000000000000000000000000000000000000000000010a6170746f735f636f696e094170746f73436f696e00 },
                             ),
-                            Modification(e87ec6a47e8d030000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001),
+                            Modification(e77ec6a47e8d030000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001),
                         ),
                     ],
```

Comment:
Those files are gone once you rebase. Please be careful to not accidentally re-introduce them.